core/state_processor.go

```diff
@@ -66,12 +69,12 @@ func (p *StateProcessor) Process(block *types.Block, statedb *state.StateDB, cfg
 	)
```

Comment:
Nit: Let's cache the config in a var and re-use across the function

Reply:
done